### DISCUSSION TITLE
Is it possible to disable the active tab indicator line?

### DISCUSSION THREAD

**Discussion by ghost** [black7375/Firefox-UI-Fix#727] [Q&A] [answered]
Title says it all, I'm currently using, the photon theme and I love it, but I would like to disable the little blue line on the top that shows that the tab is active. I've tried various solutions but they haven't worked (I'm in a hurry so I can't describe specifically though I will later if asked), so any help will be appreciated.

**Comment by black7375** [comment, marked as answer]
Try to [edit `user.js` file](https://github.com/black7375/Firefox-UI-Fix/wiki/Tutorial#edit-userjs-file)
- `userChrome.tab.photon_like_contextline` to `false`

**Reply by ghost** [reply]
Thank you this worked.
